client/src/v_video.cpp

```diff
@@ -245,10 +245,27 @@ CVAR_FUNC_IMPL(vid_pillarbox)
 		V_ForceVideoModeAdjustment();
 }
 
+//
```

Comment:
This doesn't seem related and should probably be in a separate PR, but I realize how annoying that bug is.